src/Database/LSMTree.hs

```diff
@@ -531,7 +531,10 @@ closeSession (Session session) =
 {- |
 Run an action with access to an empty table.
 
-The worst-case disk I\/O complexity of this operation is \(O(1)\).
+The worst-case disk I\/O complexity of this operation depends on the merge policy of the table:
+
+['LazyLevelling']:
+    \(O(T \log_T \frac{n}{B})\).
```

Comment:
The `n` can be slightly confusing here. Maybe it would help to mention that it refers to the size of the table at the end of the action that is performed on the empty table?